atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -201,6 +190,59 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+        def check_if_all_same(labels_to_check, source):
+            if labels_to_check:
+                label_value = source[next(iter(labels_to_check))]
+
+                for equal_name in labels_to_check:
+                    if label_value != source[equal_name]:
+                        return False
+            return True
+
+        def set_missing_labels(labels_set, all_labels, value_from, not_in=[], not_value=None):
+            labels_to_set = all_labels.difference(labels_set)
+            label_value = value_from[next(iter(labels_set))]
+            for set_label in labels_to_set:
+                if set_label in not_in and label_value == not_value[set_label]:
+                    self.log.debug("skipping label %r because it is set correctly in base image",
+                                   set_label)
+                else:
+                    self.labels[set_label] = label_value
+                    self.log.warning("adding equal label %r with value %r", set_label, label_value)
+
+        fail_build = False
+        for equal_list in Labels.LABEL_NAMES_EQUAL.values():
+            all_equal = set(equal_list)
+            found_labels_base = set()
+            found_labels_new = set()
+            for equal_label in equal_list:
+                if equal_label in new_labels:
+                    found_labels_new.add(equal_label)
+                elif equal_label in base_labels:
+                    found_labels_base.add(equal_label)
+
+            if found_labels_new:
+                if not check_if_all_same(found_labels_new, new_labels):
+                    self.log.error("labels in dockerfile don't have same values %s", equal_list)
+                    fail_build = True
+                    continue
+
+                if not fail_build:
```

Comment:
yes, and I don't want to set anymore anything once I've find out that something is wrong